docusaurus/docs-internals/editor-controls/multivalue-controls/corners.md

```diff
@@ -0,0 +1,164 @@
+
+Example of grouped control:<br/>
+![CornersGrouped](/img/multivalue-controls/cornerGrouped.png)<br/>
+![CornersGrouped](/img/multivalue-controls/cornerGroupedSecond.png)<br/><br/>
+Example of ungrouped control:<br/>
+![CornersUngrouped](/img/multivalue-controls/cornerUngrouped.png)
+
+| Name            | Type                                     | Default | Description                                                                                                                                                                                                                      |
+| :-------------- | :--------------------------------------- | :-----: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `id`            | `string`                                 |    -    | The identifier of the key where the corners will save your data                                                                                                                                                                  |
+| `type`          | `string`                                 |    -    | Type should be `"corners"` to use this control                                                                                                                                                                                   |
+| `label?`        | `string`                                 |    -    | The label displayed on the left side of the control                                                                                                                                                                              |
+| `position?`     | `number`                                 |    -    | The position of the control in toolbar                                                                                                                                                                                           |
+| `devices?`      | `"all"` \| `"desktop"` \| `"responsive"` | `"all"` | Define the devices where the control will be rendered. `"all"` renders the control on all devices. `"desktop"` renders the control only on desktop devices. `"responsive"` renders the control on both tablet and mobile devices |
+| `disabled?`     | `boolean`                                | `false` | Configure the condition under which the control is disabled or enabled                                                                                                                                                           |
+| `config?.units` | `Array<Unit>`                            |  `[ ]`  | The units of measurement displayed next to the slider value.<br/> Accepts CSS units (`"px"`, `"%"`) <br/> <br/> <b>`Unit = "px" \| "%"`</b>                                                                                      |
```

Comment:
add default for units
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/85f74e97-0b45-4188-92a9-a7d110aa7b16)